code/WorkInProgress/telescope/quantumTelescope.dm

```diff
@@ -79,6 +79,7 @@ TODO: Enforce ping rate limit here as well in case someone futzes with the javas
 		return
 
 	Topic(href, href_list)
+		if (get_dist(src, usr) > 1) return
```

Comment:
```suggestion
		if (get_dist(src, usr) > 1) 
		  using = null
		  return
```
Set `using` to null so someone else can use it.

Reply:
With this, if you walk away from the console, poke the map, then walk back within range of the console, you can no longer interact with it until you click the console again, which isn't the way other machines work and feels weird to me. How about a call to bootUser() instead and just forcing the window to close?

Reply:
Good point, that would feel weird. I think you're right, `bootUser()` is more understandable.